model-optimizer/mo/main.py

```diff
@@ -281,6 +290,32 @@ def emit_ir(graph: Graph, argv: argparse.Namespace):
         t = tm.Telemetry()
         t.send_event('mo', 'offline_transformations_status', message)
 
+        # if IR wasn't produced by offline_transformations step we need to fallback to IR
+        # produced by prepare_ir. This IR needs to be renamed from XXX_tmp.xml to XXX.xml
+        suffixes = [".xml", ".bin", ".mapping"]
+        if return_code != 0:
+            print("[ WARNING ] Using fallback to produce IR")
```

Comment:
Use the following approach to print message:

`log.error("Using fallback to produce IR.", extra={'is_warning': True})`

Reply:
Done.